cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -27,7 +27,7 @@ pub struct ParsedQObject {
     /// This is the RustObj struct that stores the invokables for the QObject
     pub rust_struct: Option<ItemStruct>,
     /// This is the Signals enum that defines the Q_SIGNALS for the QObject
```

Comment:
comment needs to be updated, this is no longer an enum

Reply:
It's the representation of the enum :thinking:

Reply:
Changed the wording a bit.